src/intrinsics/ecma262/Object.js

```diff
@@ -495,11 +495,27 @@ export default function(realm: Realm): NativeFunctionValue {
   });
 
   // ECMA262 19.1.2.9
-  if (!realm.isCompatibleWith(realm.MOBILE_JSC_VERSION) && !realm.isCompatibleWith("mobile"))
-    func.defineNativeMethod("getOwnPropertySymbols", 1, (context, [O]) => {
+  if (!realm.isCompatibleWith(realm.MOBILE_JSC_VERSION) && !realm.isCompatibleWith("mobile")) {
+    let getOwnPropertySymbols = func.defineNativeMethod("getOwnPropertySymbols", 1, (context, [O]) => {
+      if (O instanceof AbstractValue && realm.isInPureScope()) {
+        let obj = O instanceof AbstractObjectValue ? O : To.ToObject(realm, O);
+
+        return ArrayValue.createTemporalWithWidenedNumericProperty(
+          realm,
+          [getOwnPropertySymbols, obj],
+          createOperationDescriptor("UNKNOWN_ARRAY_METHOD_CALL")
```

Comment:
why `unknown_array_method_call` shouldn't we change this to `unknown_object_method_call` or something?

Reply:
nevermind, misread this part